lib/coll_8_api/operations/cancel_shipment.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+#
+# POST {{ base_url }}/api/shipment/{trackingNumber}/cancel
+#
+module Coll8Api
+  module Operations
+    class CancelShipment < Operation
+      SERVICE_NAME = 'shipment'
```

Comment:
It seems to me that having 
```ruby
SERVICE_NAME = 'shipment'
```
and
```ruby
def service
  SERVICE_NAME
end
```
adds unnecessary complexity. The string could be simply hardcoded in `#endpoint`, i.e.
```ruby
def endpoint
  "#{base_url}/shipment/#{tracking_number}/cancel"
end
```

Reply:
Yes, I will remove that as the endpoints as specialized anyway.